test/functional/eval/environ_spec.lua

```diff
@@ -17,3 +22,63 @@ describe('environment variables', function()
     eq(0, exists('$DOES_NOT_EXIST'))
   end)
 end)
+
+describe('empty $HOME', function()
+  local original_home = ''
```

Comment:
seems like this could be:

```suggestion
  local original_home = os.getenv('HOME')
```

then the `before_each` isn't needed.

Reply:
Done